### DISCUSSION TITLE
GrapesJS click plugin (no more drag-and-drop)

### DISCUSSION THREAD

**Discussion by bgrand-ch** [GrapesJS/grapesjs#5587] [Show and tell]
Hello,

For a personal project and at work, I needed to rethink the UI/UX of GrapesJS to look like [LeadPages](https://www.leadpages.com/platform/landing-page-builder) builder.

Here's one of the plugins I used:
https://github.com/bgrand-ch/grapesjs-click

Full demonstration:
https://github.com/bgrand-ch/grapesjs-click/blob/main/src/example.ts

Playground (Vue 3):
https://github.com/bgrand-ch/grapesjs-click/tree/main/playground/vue

**Comment by bgrand-ch** [comment]
See also https://github.com/GrapesJS/grapesjs/discussions/5586

**Reply by ronaldohoch** [reply]
Awesome!

**Reply by micker** [reply]
Yep

**Comment by artf** [comment]
Awesome thing @bgrand-ch Thanks for sharing ❤️

**Comment by bgrand-ch** [comment]
ℹ️ Vue playground updated
